### PR TITLE
7740 verification of docker image ipfs.

### PR DESCRIPTION
**Pull request by bthaile**
updated README to give seadragon IPNS url
updated docker to use newer base image
updated deploy doc to use -t tags for clarity and convenience.

[Clubhouse Story](https://app.clubhouse.io/augur/story/7740)

updated task with findings. 

## Submitter checklist
- [ ] Test covering functionality added/fixed in
- Check the linked story includes the following:
  - [ ] Steps to reproduce
  - [ ] Screenshot (If applicable)

## Reviewer Checklist
- [ ] Test/lint pass 
- [ ] Post merge, story marked complete

### CODE REVIEW

File: docs/deploy-production.md [hunk 1]
```diff
@@ -60,21 +60,21 @@ The same dockerfile describe above is used to host a ipfs node. Side note, if yo
 
     git clone https://github.com/AugurProject/augur.git
     cd <augur repository>
-    docker build .
+    docker build . -t augurclient
 
 After docker image has been built successfully, check for message `Successfully built <image hash>`, run it with -p to map port to the internal docker container port, ipfs swarm will serve up files via port 4001. Optionally port 8001 uses nginx to pass requests to ipfs gateway hosted in the docker container. 
 
-    docker run -p 4001:4001 -p 8001:8001 <image hash>
```
Comment (epheph) [inline docs/deploy-production.md]: Was the `-t` necessary here or was it a copypasta from `docker build`

Comment (tinybike) [reply docs/deploy-production.md]: Also curious about this, do we need to include -t augurclient any time this image is used?

**Comment by tinybike** [comment]
@bthaile does any of the IPFS docker stuff get auto-updated, or do we just need to follow the deploy-production.md doc to deploy to IPFS every time?  Asking b/c dev.augur.net is currently auto-updated from merges to mainline; does IPFS get updated at the same time or no?

**Comment by tinybike** [comment]
@bthaile I'm trying to use https://gateway.ipfs.io/ipfs/QmfZeXegvQkpUp2QGgx8LMufdyVuKSZ38e8EJuXHAS55ru/ and I get a 404 error.  Have you tested this?  What is the strategy for keeping it pinned, are you just pinning it locally?

**Comment by bthaile** [comment]
There are a few ideas here:

@epheph  yes, the ipfs docker gets updated as part of the seadragon build.  The 'AUGUR-UI-IPFS' service cranks up and hosts the docker that pins the files.

@tinybike 
There are 2 ways to pull the IPFS version of UI, one through ipfs hash or by ipns hash.

1. The url you tried above might be a old augur build directory hash therefore it isn't pinned anymore, since seadragon is rebuilt often.
Currently we don't post the augur build directory hash anywhere. It is in the output logs of the AUGUR-UI-IPFS docker container. We don't post it anywhere because the ipns hash is always available as long as the docker container is pinning, which it is. 

2. Use IPNS to host it, the auto build updates the IPNS hash to point to the new build directory hash.
https://ipfs.io/ipns/QmQBVeHLXPB7ifmCWZUw9PDktUoTEJfQLpR3bSVUgZgLR8/ 
Here is seadragon's ipns url, this stays constant. It just takes many minutes to come up. I put the IPNS url in the updated README.md.

I think we need to have IPFS/IPNS in order to have a fully distributed application, but should focus on other convenient ways users can run the UI.

File: docs/deploy-production.md (hunk 1 above)
Comment (bthaile) [reply docs/deploy-production.md]: true '-t' isn't needed (copy paste issue), it's a valid flag but for `Allocate a pseudo-TTY` from the documentation. confusing I'll remove it.

**Pull request merged**